wislfp/lib/semantics/wislCMemory.ml

```diff
@@ -0,0 +1,94 @@
+open Gillian.Concrete
+module Literal = Gillian.Gil_syntax.Literal
+
+type init_data = unit
+type vt = Values.t
+type st = Subst.t
+type err_t = unit [@@deriving show]
+type t = WislCHeap.t
+type action_ret = (t * vt list, err_t) result
+
+let init = WislCHeap.init
+let copy = WislCHeap.copy
+let pp fmt h = Format.fprintf fmt "%s" (WislCHeap.str h)
+let pp_err _fmt () = ()
+
+(* Small util for retrocompat *)
+let vstr v = Format.asprintf "%a" Values.pp v
+
+(* GetCell takes one argument, which supposedly evaluates to a pointer *)
```

Comment:
Note that right now wisl concrete execution is broken, you'd have to implement concrete load/store etc. There's probably no permissions